src/Compilers/CSharp/Test/Emit3/FirstClassSpanTests.cs

```diff
@@ -28,7 +29,7 @@ public static TheoryData<LanguageVersion> LangVersions()
     }
 
     private sealed class CombinatorialLangVersions()
-        : CombinatorialValuesAttribute(LangVersions().Select(d => d.Single()).ToArray());
+        : CombinatorialValuesAttribute(((IEnumerable<object[]>)LangVersions()).Select(d => d.Single()).ToArray());
```

Comment:
For reference: Previously `TheoryData<T>` was an `IEnumerable<object[]>`, but now it's both `IEnumerable<object[]>` and `IEnumerable<T>`, causing Linq's Select to not be resolved properly without the cast.